meeting-notes/2022-02-09.md

```diff
@@ -0,0 +1,102 @@
+    - "how to contrubute" -> to help reduce load on the discord server
+    - Briggs: do we need to copy-paste changes? 
+        - Eric: yes, would need to be copy pasted on all duplicates
+        - Briggs: Instead, use it in one place and only keep specific differences on individual repos
+            - Could use a fresh repo, or the TOP site
+        - Eric will think on it
+    -  Will be linked in the PR templates and issue templates
+    -  PR template isn't much different than current one
+    -  Issue templates 
+        - has default labels and other items
+        - Important: There is a checkbox for being assigned an issue if they create it
+        - Briggs: do we want to change verbage on checkbox to "I am working on this issue"
+            - Maybe? The reason for not changing it is to prevent people from woking on things that we may not want
+            - Sully: disagree, morale issues with letting people work on things and denying them
+                - Briggs: want to ensure that we don't lose velocity because of our slowness to get to an issue
+            - Daniel: personality types could be an issue, some will just just do it, some will want to wait
+            - Daniel: Change to "I would like to be assigned this issue to work on it"
+                - Agressive people will work on it anyway
+                - Hesitent people will wait for approval
+            - Changes got merged into bug_report.md 
+    - TL;DR, this is for consistancy throughout the repos to make everything easier for people
+    - Action Items: 
+        - Eric: Move content to a central repo and link from supporting repos with nessessary modifications
+
+- Bring up idea to "improve link text throughout the curriculum" [Zach/Eric/Sully]
+    - Improve the existing link text in the curriculum to be more accessible
+    - Potentially create a large opportunity like the knowledge checks overhaul
+    - Give more meaningful names to links
+        - instead of "click [here](here)", say "check out [MDN's article on closure]()"
+    - This also ties into a11y issues
+        - Sully: should we wait and do a full a11y pass?
+        - two options
+            - have it be part of the overall style project as it's tied into re-arranging links
+            - community could be involved with updating link text as well as putting it in other sections
+        - Eric will do this with the style guide thing
+    - Plans on focusing on Foundations at first, then will focus on the courses in bursts
+    - Briggs: May want to avoid updating lessons
+        - Eric: important to update "NOW" rather than later
+        - Worry about it when we get to that issue, foundations is a good start
+
+- Discuss suggestions on club-40 mini-modding issue (<5 minutes) [Daniel]
+    - Under Rule 1 actually state that community members should not attempt to moderate the chat themselves
+    - Under Rule 1 indicate who moderators/maintainers actually are => (red/blue/orange)
```

Comment:
ummmm excuuuuuse me.  core is _gold_ not orange.  i take offense at this 😠 

![lol teasin](https://media.giphy.com/media/VEtou2u585yGk/giphy.gif)